lib/gdal/extensions/raster_band/extensions.rb

```diff
@@ -40,7 +40,7 @@ def to_na(to_data_type = nil)
       end
 
       def to_nna
-        Numo::NArray[to_a]
+        NArray.to_na(to_a)
```

Comment:
`#to_na` is supposed to return an NArray, but `#to_nna` is supposed to return a Numo::NArray.  (it's unfortunate that the return value wasn't documented)

Reply:
Oh now it all makes sense